documentation/modules/managing/proc-setting-broker-limits.adoc

```diff
@@ -20,25 +20,21 @@ The total limit is distributed across all clients accessing the broker.
 For example, you can set a byte-rate threshold of 40 MBps for producers.
 If two producers are running, they are each limited to a throughput of 20 MBps.
 
-Storage quotas throttle Kafka disk storage limits between a soft limit and hard limit.
-The limits apply to all available disk space.
-Producers are slowed gradually between the soft and hard limit.
-The limits prevent disks filling up too quickly and exceeding their capacity.
+Storage quotas enforce throttling on Kafka disk storage when it reaches the specified limit.
+You can specify the limit in bytes or percentage of available disk space.
+The limit applies to all available disk space across all brokers.
+It prevents disks filling up too quickly and exceeding their capacity.
```

Comment:
I think this is misleading. The limits apply to every disk individually. No `accross all brokers` or against `available disk space`.